scripts/module/debug.nut

```diff
@@ -12,8 +12,19 @@ function onPlayerCommand( player, cmd, text ) {
     ReloadScripts();
   } else if (cmd == "pos") {
     print(player.Pos);
+  } else if (cmd == "gtp" || cmd == "gotopos") {
+    if (text) {
+      local coords = split(text, " ");
+      if (IsNum(coords[0]) && IsNum(coords[1]) && IsNum(coords[2])) {
+        local pos = Vector(coords[0].tointeger(), coords[1].tointeger(), coords[2].tointeger());
+        TeleportPlayer(player, pos, 0);
+        MessagePlayer("Teleported to (" + text + ").", player);
+      } else {
+        MessagePlayer("[#ff0000]Usage: /" + cmd + " X Y Z", player);
+      }
+    }
   } else if (cmd == "help") {
-    MessagePlayer("[#8080A0]/r[eload] /pos", player);
+    MessagePlayer("[#8080A0]/r[eload] /pos /g[o]t[o]p[os]", player);
```

Comment:
yikes.. this is ugly. How about `set_pos` and `get_pos` instead?

Generally: You could also extend the `/t[eleport]` command to accept coordinates (and possibly restrict that feature to admins)

If we implement #15 we also don't have to worry about long command names anymore anyway